src/parsers/transactions.ts

```diff
@@ -333,8 +332,7 @@ const parseTransaction = async (
   txn = overrideTradeRefund(txn);
 
   if (txn.changes.length) {
-    const internalTransactions = map(
-      txn?.changes,
+    const internalTransactions = txn.changes.map(
```

Comment:
```suggestion
    const internalTransactions = txn?.changes.map(
```

Reply:
Also it might need a default value for the result array

Reply:
on top was a check which ensures that it is an array with at least one element
`if (txn.changes.length)`